Public/Src/Tools/ServicePipDaemon/ServicePipDaemon.cs

```diff
@@ -82,6 +87,19 @@ protected enum DaemonCounter
 
         #region Options and commands 
 
+        internal static readonly StrOption ConfigFile = RegisterDaemonConfigOption(new StrOption("configFile")

```

Comment:
Why do we expect that every daemon run by service pip has a configuration file? Let say someone else creates a daemon tool, and wants to use our service pip?

Reply:
We do not 'expect' nor 'require' that derived classes use config files. Options listed here are 'supported' options. 

---
In reply to: [313638353](https://github.com/microsoft/BuildXL/pull/727#discussion_r313638353) [](ancestors = 313638353)